build.cmd

```diff
@@ -1,119 +1,23 @@
-  call set unprocessedBuildArgs=%%__args:*!processedArgs!=%%
-)
-
-:Next
-shift /1
-goto Loop
-
-:Tools
-:: Setup VS
-if not defined VisualStudioVersion (
-    if defined VS140COMNTOOLS (
-        call "%VS140COMNTOOLS%\VsDevCmd.bat"
-        goto :Build
-    )
-
-    echo Error: build.cmd requires Visual Studio 2015.
-	:: *** start WCF Content ***
-    echo        Please see https://github.com/dotnet/wcf/blob/master/Documentation/developer-guide.md for build instructions.
-	:: *** end WCF Content ***
-    exit /b 1
-)
+if /I [%1] == [-?] goto Usage
 
 :Build
-:: Restore the Tools directory
-call %~dp0init-tools.cmd
-
-:: Call the builds
-if "%__buildSpec%"=="managed"  goto :BuildManaged
-
-:BuildNative
-:: Run the Native Windows build
-echo [%time%] Building Native Libraries...
-:: Generate Native versioning assets
-set __binDir=%~dp0bin
-set __versionLog=%~dp0version.log
-if not exist "%__binDir%\obj\_version.h" (
-    msbuild "%~dp0build.proj" /nologo /t:GenerateVersionHeader /p:GenerateNativeVersionInfo=true > "%__versionLog%"
-)
-IF EXIST "%~dp0src\native\Windows\build-native.cmd" (
-    call %~dp0src\native\Windows\build-native.cmd %__args% >nativebuild.log
-    IF ERRORLEVEL 1 (
-        echo Native component build failed see nativebuild.log for more details.
-    ) else (
-        echo [%time%] Successfully built Native Libraries.
-    )
-)
-
-:: If we only wanted to build the native components, exit
-if "%__buildSpec%"=="native" goto :eof 
-
-:BuildManaged
-:: Clear the 'Platform' env variable for this session,
-:: as it's a per-project setting within the build, and
-:: misleading value (such as 'MCD' in HP PCs) may lead
-:: to build breakage (issue: #69).
-set Platform=
-
-:: Log build command line
-set _buildproj=%~dp0build.proj
-set _buildlog=%~dp0msbuild.log
-set _binclashLoggerDll=%~dp0Tools\net45\Microsoft.DotNet.Build.Tasks.dll
-set _binclashlog=%~dp0binclash.log
-set _buildprefix=echo
-set _buildpostfix=^> "%_buildlog%"
-call :build %__args%
-
-:: Build
-set _buildprefix=
-set _buildpostfix=
-echo [%time%] Building Managed Libraries...
-call :build %__args%
-
-goto :AfterBuild
-
-:build
-%_buildprefix% msbuild "%_buildproj%" /nologo /maxcpucount /v:minimal /clp:Summary /nodeReuse:false /flp:v=normal;LogFile="%_buildlog%";Append /flp2:warningsonly;logfile=%~dp0msbuild.wrn /flp3:errorsonly;logfile=%~dp0msbuild.err "/l:BinClashLogger,%_binclashLoggerDll%;LogFile=%_binclashlog%" !unprocessedBuildArgs! %_buildpostfix%
-set BUILDERRORLEVEL=%ERRORLEVEL%
-goto :eof
-
-:AfterBuild
+call %~dp0build-managed.cmd %*
+exit /b %ERRORLEVEL%
 
+:Usage
 echo.
-:: Pull the build summary from the log file
-findstr /ir /c:".*Warning(s)" /c:".*Error(s)" /c:"Time Elapsed.*" "%_buildlog%"
-echo [%time%] Build Exit Code = %BUILDERRORLEVEL%
-
-exit /b %BUILDERRORLEVEL%
\ No newline at end of file
+echo There are new changes on how we build. Use this script only for generic
+echo build instructions that apply for both build native and build managed.
```

Comment:
Things have changes sufficiently that I am inclined to simplify and follow Mariana's advice. I'm not for this PR mainly because I want to get VSO working as quickly as possible with these fixes which will be easier by using the CoreFx build definitions as a guide.